templates/frontend/pages/indexJournal.tpl

```diff
@@ -28,6 +28,12 @@
 		</div>
 	{/if}
 
+    {* Journal Description *}
+    {if $currentContext->getLocalizedData('description')}
+        <h2>{translate key="about.aboutContext"}</h2>
```

Comment:
Let's wrap this whole section in a div:

```
<div class="homepage_about">
  ...
</div>
```

Journals often use CSS code to modify default theme, so wrapping it in a div with a unique class makes it easier for them to target the description for changing the style if they want.

When you do this, you'll notice the margins between the description and the announcements gets collapsed. You'll probably need to add some CSS code to get the spacing correct. You can add that to `plugins/themes/default/styles/pages/indexJournal.less`.

(Less is similar to Sass, with a slightly different syntax. You can read up on it at: http://lesscss.org/)

Reply:
Got that done, plus the theme option!